tests/functional_tests.py

```diff
@@ -67,11 +76,23 @@ def test_redirect(self):
 		loc = req.headers.get("location")
 		self.assertEqual(loc, self.shortDest)
 
+	def test_redirect_head(self):
+		"""Test basic redirecting capability of schort for head requests"""
+		self.assertNotEqual(len(self.shortID), 0)
```

Comment:
This is already implicit tested by `test_redirect`. It could never pass (return 301) if the previously generated `shortID` would be invalid.